package.xml

```diff
@@ -1,27 +1,53 @@
 <?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="3">
   <name>odom_transform</name>
   <version>0.0.0</version>
-  <description>pose in imu frame to body frame transform</description>
+  <description>TODO: Package description</description>
+  <maintainer email="yangzhou@nyu.edu">Yang Zhou</maintainer>
+  <maintainer email="ys5153@nyu.edu">Yash Shetye</maintainer>
+  <maintainer email="jx1190@nyu.edu">Jiuhong Xiao</maintainer>
+  <license>TODO: License declaration</license>
 
-  <author email="cw4030@nyu.edu">Chenyu</author>
-  <maintainer email="cw4030@nyu.edu">Chenyu</maintainer>
+  <!-- ROS1: Dependencies needed to compile this package. -->
+  <buildtool_depend condition="$ROS_VERSION == 1">catkin</buildtool_depend>
+  <depend condition="$ROS_VERSION == 1">cmake_modules</depend>
+  <depend condition="$ROS_VERSION == 1">roscpp</depend>
+  <depend condition="$ROS_VERSION == 1">roslib</depend>
+  <depend condition="$ROS_VERSION == 1">rosbag</depend>
+  <depend condition="$ROS_VERSION == 1">tf</depend>
+  <depend condition="$ROS_VERSION == 1">std_msgs</depend>
+  <depend condition="$ROS_VERSION == 1">sensor_msgs</depend>
+  <depend condition="$ROS_VERSION == 1">nav_msgs</depend>
+  <depend condition="$ROS_VERSION == 1">cv_bridge</depend>
```

Comment:
I don't think we used opencv

Reply:
Yes we don't need it, previously due to some errors I encountered I had added it thinking it will solve the errors but now I have tested it, it's not required, thanks for pointing it out, will remove it.